ios/BetterRailWidget/Live Activity/Views/CircularProgressView.swift

```diff
@@ -23,13 +29,23 @@ struct CircularProgressView: View {
     }
   }
   
+  var minutesLeft: Int {
+    getMinutesLeft(targetDate: end)
+  }
+  
   var tintColor: Color
   
   var body: some View {
     ZStack {
       ProgressView(timerInterval: start...end, countsDown: false, label: {}, currentValueLabel: { EmptyView() }).progressViewStyle(.circular).tint(tintColor)
-      Image(systemName: iconName).font(.system(size: 8.0))
-        .scaleEffect(x: vm.isRTL ? -1 : 1, y: 1, anchor: .center)
+      
+      if content == .time && vm.context.attributes.frequentPushesEnabled && minutesLeft < 100 {
+        Text(String(minutesLeft))
+          .font(.system(size: 11, weight: .semibold))
```

Comment:
Did you check how the rounded style looks here? Might be nicer

Reply:
I checked with multiple combinations of font sizes, styles and weight. That's the best looking one (especially next to the time) IMO.